src/app/components/search/SearchResultsTable/LinkedItemsCountCell.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import NumberCell from './NumberCell';
 
 export default function LinkedItemsCountCell({ projectMedia }) {
-  const linkedItemsCount = projectMedia.linked_items_count;
-  return <NumberCell value={linkedItemsCount} />;
+  const { related_count } = projectMedia;
```

Comment:
This file should be renamed to `RelatedCountCell.js`.

Also ... if this is `RelatedCount`, should the other one `RequestsCount`? What's the different between a "request" and a "request related"?

Reply:
we have 2 types of request counts: requests made on the item itself (`requests_count`), and requests made on the item + all its related items (`requests_related_count`)